src/event.rs

```diff
@@ -9,7 +9,7 @@ use self::Event::*;
 pub enum Event {
     AddNode(Name),
     RemoveNode(Name),
-    RemoveNodeFrom(Prefix),
+    RemoveNodeFrom(Prefix), 
```

Comment:
Suddenly - trailing whitespace (I'm guessing that's just rustfmt acting up, though).